core/tauri/src/endpoints/shell.rs

```diff
@@ -25,7 +25,7 @@ type ChildId = u32;
 type ChildStore = Arc<Mutex<HashMap<ChildId, crate::api::process::CommandChild>>>;
 
 #[cfg(any(shell_execute, shell_sidecar))]
-fn command_childs() -> &'static ChildStore {
+fn command_child_store() -> &'static ChildStore {
```

Comment:
Not at all certain about this change. Normally it's `child` -> `children`. Unless you're saying `child`+ possessive `'s` which seemed unlikely.